src/main/java/com/epam/demo/StringUtils.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.demo;
+
+
+import org.apache.commons.lang3.math.NumberUtils;
+
+public class StringUtils {
+
+    public static boolean isPositiveNumber(String str) {
```

Comment:
The method is public accessible in your library, please provide a javadoc comment how to use it